PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,12 +1,24 @@
-<!--- Provide a general summary of your changes in the Title above -->
-
-<!-- 
-     Make sure you have read the WIKI STANDARDS before you submit a PR that changes, adds or removes a wiki page.
-     https://www.azerothcore.org/wiki/wiki-standards 
+<!---
+Provide a general summary of your changes in the Title above.
+Make sure you have read the WIKI STANDARDS before you submit a PR that changes, adds or removes a wiki page.
+English: https://www.azerothcore.org/wiki/wiki-standards
+Spanish: https://www.azerothcore.org/wiki/es/wiki-standards
 -->
 
 ### Description
 
+- 
+- 
+
 ### Related Issue
 
 Closes
+
+## Thank you for contributing to the AzerothCore wiki.
+
+If you are making a change to a file that requires an update to Spanish or you are creating one, please, if you can, within the pull request or the chat itself, tag or mention **@pangolp** so that he can create/update the file to Spanish as well. Thank you.
```

Comment:
I'd suggest making this a comment or you will be spammed af :P

Reply:
Also technically it would apply for any non Spanish PR

Reply:
We'll put it as a comment then.